tests/core/test_policies.py

```diff
@@ -379,28 +420,42 @@ def create_policy(
         resource: Resource,
         execution_context: ExecutionContext,
         config: Optional[Dict[Text, Any]] = None,
-    ) -> Policy:
-        max_history = None
-        if isinstance(featurizer, MaxHistoryTrackerFeaturizer):
-            max_history = featurizer.max_history
-        return MemoizationPolicy(priority=priority, max_history=max_history)
+    ) -> EitherPolicy:
+        config = config or MemoizationPolicyGraphComponent.get_default_config()
+        config[POLICY_PRIORITY] = priority
+        return MemoizationPolicyGraphComponent(
+            config, model_storage, resource, execution_context, featurizer
+        )
 
-    def test_featurizer(self, trained_policy: Policy, tmp_path: Path):
+    def test_featurizer(
+        self,
+        trained_policy: EitherPolicy,
+        resource: Resource,
+        model_storage: ModelStorage,
+        tmp_path: Path,
+        execution_context: ExecutionContext,
+    ):
         assert isinstance(trained_policy.featurizer, MaxHistoryTrackerFeaturizer)
         assert trained_policy.featurizer.state_featurizer is None
-        trained_policy.persist(str(tmp_path))
-        loaded = trained_policy.__class__.load(str(tmp_path))
+        config = self._config(DEFAULT_POLICY_PRIORITY)
+        trained_policy.persist()
```

Comment:
shouldn't be required as the policy should already be persisted during training
```suggestion
```

Reply:
Is that really the new default behavior, i.e. to persist automatically when trained? If so I haven't implemented it yet!

Reply:
The fixture isn't doing persistence either

Reply:
> Is that really the new default behavior, i.e. to persist automatically when trained? If so I haven't implemented it yet!

yep (see e.g. `TEDPolicy` or `DIETClassifier`)